docs/dev/types_serialization.md

```diff
@@ -86,11 +86,11 @@ val input = env.fromElements(
     WordCount("hello", 1),
     WordCount("world", 2)) // Case Class Data Set
 
-input.keyBy("word")// key by field expression "word"
+input.keyBy(_.word) // .keyBy("word") is @Deprecated
```

Comment:
In my opinion, there is no point in mentioning something that is deprecated. Let's just drop the comment.